scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -149,6 +149,11 @@ public static BuildTargetResult RestoreLockedCoreHost(BuildTargetContext c)
             return c.Success();
         }
 
+        [Target(
+            nameof(CompileCoreHost),
+            nameof(BuildCoreHostPackages))]
+        public static BuildTargetResult BuildCoreHost(BuildTargetContext c) => c.Success();
```

Comment:
Its the current state of the repo. Keep `CompileCoreHost` and `BuildCoreHostPackages` as unrelated targets, similar to `CompileCoreHost` and `PackagePkgProjects` before this change.

The original PR - https://github.com/dotnet/cli/pull/2439